doc/feature/restAPI.md

```diff
@@ -0,0 +1,56 @@
+Publishing the REST API exposed by the HTTP Server
+============
+
+The http server exposes C++ objects via a REST API, in a JSON serialized format.
+The purpose of this document is to describe how this API request is implemented,
+and invoked.
+
+## Requirements
+
+* Define how to invoke the application REST API
+* Implement the JSON representation of the REST API
+
+## API
+
+Invoking a GET resquest on the /api url returns a JSON representation of the
+schemas for all registered serializables.
```

Comment:
Can you make this an issue, discuss alternatives and propose a solution? I'm not against /api per se.